src/data/fin.lean

```diff
@@ -117,7 +117,7 @@ lemma ne_iff_vne (a b : fin n) : a ≠ b ↔ a.1 ≠ b.1 :=
 
 protected lemma mk.inj_iff {n a b : ℕ} {ha : a < n} {hb : b < n} :
   (⟨a, ha⟩ : fin n) = ⟨b, hb⟩ ↔ a = b :=
-⟨subtype.mk.inj, λ h, by subst h⟩
+subtype.mk_eq_mk
```

Comment:
Is this lemma actually a duplicate?  Should it be made an alias?

Reply:
Pretty much every subtype is full of "duplicates" like this - but this isn't an alias, because the definition is in terms of `fin` and not `subtype _`; they're defeq but not syntactically equal.